Update manage-savings-plan.md (#125327)
Azure/azure-content: articles/cost-management-billing/savings-plan/manage-savings-plan.md

## Change Billing subscription
Currently, the billing subscription used for monthly payments of a savings plan can't be changed.

> ## Check billing subscription for an Azure Savings Plan

Reviewer: ```suggestion
## Check billing subscription for an Azure savings plan
```

To check the billing subscription for an Azure savings plan:

1. Sign in to the [Azure portal](https://portal.azure.com).
2. Select **All services** > **Savings Plan**.
3. Select the Savings Plan.
4. Select **Renewal**.

> 5. Select **Replace this savings plan with a new savings plan purchase**

Reviewer: ```suggestion
5. Select **Replace this savings plan with a new savings plan purchase**.
```

> This will show you the billing subscription of current savings plan along with other details. You can update the details like scope, billing subscription, quantity, term, and billing frequency for the new savings plan which will be purchased automatically upon expiry of current savings plan.

Reviewer: ```suggestion
These steps show you the billing subscription of current savings plan along with other details. You can update the details like scope, billing subscription, quantity, term, and billing frequency for the new savings plan which will be purchased automatically upon expiry of current savings plan.
```
